packages/vsce/src/SshConfigUtils.ts

```diff
@@ -331,7 +351,9 @@ export class SshConfigUtils {
                 value: selectedProfile.name!.replace(/\./g, "_"),
                 validateInput: (input) => (input.includes(".") ? "Name cannot contain '.'" : null),
             });
-            const existingProfile = profiles.find((profile) => profile.name === selectedProfile.name);
+            if (!selectedProfile.name) return;
+            const existingProfile = configApi.layerActive().properties.profiles[selectedProfile.name];
+            //profiles.find((profile) => profile.name === selectedProfile.name);
             if (existingProfile) {
                 const overwriteResponse = await vscode.window.showQuickPick(["Yes", "No"], {
                     placeHolder: `A profile with the name "${selectedProfile.name}" already exists. Do you want to overwrite it?`,
```

Comment:
There is an empty else condition a few lines below 😋 

should we just remove it? 😅